packages/devtools_app/lib/src/shared/tree.dart

```diff
@@ -117,7 +117,9 @@ class _TreeViewState<T extends TreeNode<T>> extends State<TreeView<T>>
     if (widget.onItemExpanded == null && item.isExpandable) {
       item.toggleExpansion();
     }
-    await widget.onItemSelected(item);
+    if (widget.onItemSelected != null) {
+      await widget.onItemSelected!(item);
+    }
```

Comment:
I think you could avoid the `if`s and `!`s by doing something like `await widget.onItemSelected?.call(item);`

Reply:
Oh nice, done!